tests/installation/partitioning_raid.pm

```diff
@@ -211,51 +210,58 @@ sub run {
         if ($_ eq 'vdd' and get_var('UEFI')) {
             assert_screen 'partitioning_raid-disk_vdd_with_partitions-selected';
-            send_key 'left';                                                                         # fold the drive tree
+            send_key 'left';                                                                                            # fold the drive tree
             assert_screen 'partitioning_raid-hard_disks-unfolded';
-            send_key 'right';                                                                        # select first disk
+            send_key 'right';                                                                                           # select first disk
             assert_screen 'partitioning_raid-disk_vda_with_partitions-selected';
-            send_key 'alt-e';                                                                        # edit first partition
+            send_key 'alt-e';                                                                                           # edit first partition
             assert_screen 'partition-format';
-            send_key 'alt-a';                                                                        # format as FAT (first choice)
+            send_key 'alt-a';                                                                                           # format as FAT (first choice)
             assert_screen 'partitioning_raid-format_fat_UEFI';
-            send_key 'alt-o';                                                                        # mount point selection
+            send_key 'alt-o';                                                                                           # mount point selection
             assert_screen 'partitioning_raid-mount_point-focused';
-            type_string '/boot/efi';                                                                 # enter mount point
+            type_string '/boot/efi';                                                                                    # enter mount point
             assert_screen 'partitioning_raid-mount_point_boot_efi';
             send_key $cmd{finish};
             assert_screen 'expert-partitioner';
             send_key 'shift-tab';
             send_key 'shift-tab';
-            send_key 'left';                                                                         # go to top "Hard Disks" node
+            send_key 'left';                                                                                            # go to top "Hard Disks" node
             assert_screen 'partitioning_raid-hard_disks-unfolded';
-            send_key 'left';                                                                         # fold the drive tree again
+            send_key 'left';                                                                                            # fold the drive tree again
```

Comment:
That comments on the right hand side are quite far apart. Can you try to put the comments on top of each line? Or try to add in another blank line before this section, e.g. line 230 and above, and try what `tools/tidy` gives you